developing/vcs/README.md

```diff
@@ -2,7 +2,321 @@
 
 ## Overview
 
-Source control has less to do with the vcs system you use and more to do with the stages code goes through on the journey from inside a developers head, tested and merged into the main line, deployed to production and thence in time to being identified as that accursed legacy system that is causing all the problems.
+Source control has less to do with the VCS system you use, and more to do with the stages code goes through on the journey from inside a developer's head, tested and merged into the main line, deployed to production, and thence in time to being identified as that accursed legacy system that is causing all the problems.
+
+## Repo maintenance and hygiene
+
+* Avoid including files in source control that are specific to your development machine or process.
+* Ignore files that are specific to your machine or process in your
```

Comment:
Why? Any time we tell people they need to do something, we need to explain very clearly why this is important.